### PR TITLE
Make portfolio responsive and viewable on desktop devices

### PR DESCRIPTION
**Pull request by emmiiorji**
I have added responsiveness to the different sections of the portfolio website, following the given [design template2](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A1278)
Find below some screenshots of the desktop version of the website.

image

image

image

image

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -38,26 +46,33 @@ <h2>I am a Software Developer</h2>
...
 
   <section class="works-section">
-    <h2>My Recent Works</h2>
-    <hr>
+    <div id="works-header">
+      <h2>My Recent Works</h2>
+      <hr>
+    </div>
     <div class="cards-container">
       <div class="card">
```
Comment (ShoiraTa) [inline index.html:54]: - [ ] Please make sure that when the project is hovered on with the mouse the description and dark opacity disappears and a See Project button appears at the bottom of the image and takes the whole image width mentioned button should take you to the Live Demo of the project or to the GitHub repository. Currently, button that appears on the hover is missing. This is how the card should look when hovered: 
Screenshot 2022-03-31 at 19 24 03

File: index.html [hunk 2]
```diff
@@ -38,26 +46,33 @@ <h2>I am a Software Developer</h2>
...
-          <li>Ruby</li>
-        </ul>
-        <button type="button">
-          See Project
-        </button>
+        <div class="card-img">
+          <img src="./assets/img/project1-img.png" alt="Multi-Post Stories Project">
+        </div>
+        <div id="leading-card">
+          <h2>Multi-Post Stories</h2>
```
Comment (ShoiraTa) [inline index.html:59]: - [ ] Please add a little padding inside the card, currently, the content too close to the left border:

Your design  | Figma design
------------- | -------------
Screenshot 2022-04-01 at 19 06 36 | Screenshot 2022-04-01 at 19 06 20

File: index.html [hunk 3]
```diff
@@ -196,7 +211,7 @@ <h1>About me</h1>
     </div>
   </section>
 
-  <footer class="footer">
+  <footer id="footer">
     <p>I'm always interested in hearing about new projects,
```
Comment (ShoiraTa) [inline index.html:217]: - [ ] On the bottom of the footer there are social links, I can see that you made it as a part of the form. This is a bad practice, and it leads to bugs like the icons are not centered in the middle and the divider line is not reaching the right side of the document. I recommend you to add a new div for the bottom footer with icons, and align the content with flex properties making sure that the icons are centered (horizontally and verticaly), the divider stretches from the left side to the right.
Your footer:
Screenshot 2022-04-01 at 19 13 49

- [ ] Please also make sure to decrease the height of the icons container it is slightly higher than in Figma.

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @emmiiorji,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (emmiiorji) [reply index.html:54]: Hi @ShoiraTa 

Thanks for your kind and detailed review.

I have endeavoured to fix them up. However, it seems solving this current conversation will require making use of JavaScript. The project seems not to require implementing JavaScript functionalities.

File: index.html [hunk 4]
```diff
@@ -38,39 +46,48 @@ <h2>I am a Software Developer</h2>
...
 
   <section class="works-section">
-    <h2>My Recent Works</h2>
-    <hr>
+    <div id="works-header">
+      <h2>My Recent Works</h2>
+      <hr>
+    </div>
     <div class="cards-container">
       <div class="card">
```
Comment (elmar8287) [inline index.html:54]: - [ ] Please, consider fixing the previous review suggestion regarding the hovered project. To implement it without JS, using CSS could be a solution. 👍

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @emmiiorji.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (emmiiorji) [reply index.html:54]: Hi @elmar8287 

Thank you for pointing that out. I have implemented it using CSS. Can you kindly check if it meets the requirement?

image

Comment (prolajumokeoni) [reply index.html:54]: > Hi @elmar8287
> 
> Thank you for pointing that out. I have implemented it using CSS. Can you kindly check if it meets the requirement?
> 
> image

Hi @emmiiorji currently this feature is not implemented properly, just as the first reviewer requested

File: index.html [hunk 5]
```diff
@@ -38,39 +46,48 @@ <h2>I am a Software Developer</h2>
...
+          <ul>
+            <li>html</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+        </div>
+        <button type="button" id="show" class="see-project">
           See Project
         </button>
       </div>
```
Comment (prolajumokeoni) [inline index.html:93]: ### OPTIONAL
- [ ] just as the[ first reviewer](https://github.com/emmiiorji/my_new_portfolio/pull/4/files#r840678187) requested, when the project is hovered on with the mouse the description and dark opacity disappears and a `See Project` button appears at the bottom of the image
- [ ] to achieve this, you can add a display none to your button  to hide button when the page initially loads and for the description and dark opacity a display none on hover, then finally on hover you add a display block to your button, kindly research more about [this](https://www.w3schools.com/howto/howto_css_display_element_hover.asp), all cards on hover should look like your first card 👍🏿 
- Adding this hover effect to your project will be a cool feature 😉

**Review by prolajumokeoni — APPROVED** [review]
Hi @emmiiorji, 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_Check inline comments_


all-done-gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ShoiraTa) [reply index.html:54]: Hi @emmiiorji,sorry for a late response. I guess you already know by now that these changes can be done with the help of CSS

Comment (emmiiorji) [reply index.html:54]: Yes, @ShoiraTa. It was a good spur to learn about that.

Thank you once again.

File: index.html (hunk 4 above)
Comment (emmiiorji) [reply index.html:54]: Hi @prolajumokeoni.

Thank you for your response.

Please, can you specify the feature not properly implemented? However, if it has to do with linking the demo or GitHub repo of the project, the projects currently showing are only dummies.

Comment (prolajumokeoni) [reply index.html:54]: The hover effect for the cards, please check my comment below 🤗

File: index.html (hunk 5 above)
Comment (emmiiorji) [reply index.html:93]: Thank you @prolajumokeoni.

I had already done this. The screenshot shown in the comment was taken with the mouse hovering on one of the cards (the one that has the "See Project" button showing).

Comment (emmiiorji) [reply index.html:93]: Hi @prolajumokeoni

I'm sorry. I just discovered I didn't push the code from my local repo before requesting another review.

**Pull request merged**